src/components/legend/attributes.js

```diff
@@ -90,6 +90,33 @@ module.exports = {
         ].join(' ')
     },
 
+    itemclick: {
+        valType: 'enumerated',
+        values: ['toggle', 'focus', false],
+        dflt: 'toggle',
+        role: 'info',
+        editType: 'legend',
+        description: [
+            'Determines the behavior on legend item click.',
+            '*toggle* toggles the visible of the item clicked on the graph.',
```

Comment:
visible -> visibility

Reply:
done in https://github.com/plotly/plotly.js/pull/3862/commits/88e566a8abb03cb3971f4af73d7a07b2a2a11ed5